### PR TITLE
Docs tweak/update

### PR DESCRIPTION
**Pull request by daviddrysdale**
# Checklist

- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

File: docs/abi.md [hunk 1]
```diff
@@ -214,8 +211,18 @@ Fills a buffer with random bytes.
 
 The host functions described in the previous section allow opaque blobs of data
 to be exchanged (along with handles to other channels). When communicating with
-the pseudo-Nodes provided by the Oak system, these opaque blobs of data are
-defined to take the form of serialized protocol buffer messages. These include:
+the Runtime, these opaque blobs of data are defined to take the form of
+serialized protocol buffer messages:
+
+- The label values included on Node and channel creation operations are in the
+  form of a serialized [`Label`](../oak/proto/label.proto) message.
```
Comment (tiziano88) [inline docs/abi.md:218]:
```suggestion
  form of a serialized [`Label`](/oak/proto/label.proto) message.
```

File: docs/abi.md [hunk 2]
```diff
@@ -214,8 +211,18 @@ Fills a buffer with random bytes.
...
+- The Node configuration information that is include on `node_create` operations
+  is in the form of a serialized
+  [`NodeConfiguration`](../oak/proto/application.proto) message.
```
Comment (tiziano88) [inline docs/abi.md:221]:
```suggestion
  [`NodeConfiguration`](/oak/proto/application.proto) message.
```

File: docs/abi.md [hunk 3]
```diff
@@ -214,8 +211,18 @@ Fills a buffer with random bytes.
 
 The host functions described in the previous section allow opaque blobs of data
 to be exchanged (along with handles to other channels). When communicating with
-the pseudo-Nodes provided by the Oak system, these opaque blobs of data are
-defined to take the form of serialized protocol buffer messages. These include:
+the Runtime, these opaque blobs of data are defined to take the form of
+serialized protocol buffer messages:
+
+- The label values included on Node and channel creation operations are in the
```
Comment (tiziano88) [inline docs/abi.md:217]: FWIW, this section seems to repeat what is already defined in the various descriptions above, but fine to keep around as extra clarification.

File: scripts/format_doc [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (tiziano88) [inline scripts/format_doc:1]: Should this be invoked by the `format` script now?

File: docs/abi.md (hunk 1 above)
Comment (daviddrysdale) [reply docs/abi.md]: Done throughout.

File: docs/abi.md (hunk 2 above)
Comment (daviddrysdale) [reply docs/abi.md]: Done.

File: scripts/format_doc (hunk 4 above)
Comment (daviddrysdale) [reply scripts/format_doc:1]: It doesn't quite line up (`prettier` is also invoked for `.toml` and `.yaml` files) so I left them separate.

File: docs/abi.md (hunk 3 above)
Comment (daviddrysdale) [reply docs/abi.md]: That's deliberate, this section is trying to describe conventions that apply across the whole ABI surface.

However, now that it doesn't just apply to pseudo-Node communication but to core host functions too, I think this section belongs up next to the "Integer Types" section.

**Pull request merged**